index.html

```diff
@@ -211,12 +211,12 @@ <h3>Skills</h3>
     <footer>
       <h3 id="contact-info" class="contact_head">I'm always interested in hearing about new projects, so if you'd like
         to chat please get in touch.</h3>
-      <form novalidate action="https://formspree.io/f/meqnjpgd" method="post">
+      <form novalidate action="https://formspree.io/f/meqdykek" method="post">
 
         <input type="text" id="full-name" maxlength="30" name="full_name" placeholder="Full name" required>
         <input type="email" id="email" name="user-email" placeholder="Email address"
           pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
-        <textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+        <textarea name="message" maxlength="500" placeholder="Enter message here" required></textarea>
         <div class="errorButton , flex">
           <button class="form_button" type="submit">Get in touch</button>
           <small class="collapse"></small>
```

Comment:
i think you it would be better if you avoid extra div tag and I also think that you do not need any extra classes like flex and errorButton. you can use only one class name so that the code looks clean
thanks.

Reply:
Thank you for your suggestions, It helps me a lot to improve my skills